practice/batch/services/service_actions_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec'
+require_relative 'services_actions'
+
+
+describe 'Services' do
+
+  before(:each) do
+    @service_name = 'vpnagent'
+    @status_stopped = SERVICES::STOPPED
+    @status_running = SERVICES::RUNNING
+  end
+
+  it 'should be stopped and started again properly' do
+
+    if SERVICES::service_status(@service_name, @status_running)
```

Comment:
We want to make sure that the driver is running at the beginning of the test. I guess it's enough to have something like
start_service unless service_status(stopped)